internal/apps/msp/apm/service/components/resources-runtime-monitor-java/runtime/provider.go

```diff
@@ -57,7 +57,7 @@ type provider struct {
 }
 
 func (p *provider) getMemoryHeapLineGraph(ctx context.Context, startTime, endTime int64, tenantId, instanceId, serviceId string) ([]*model.LineGraphMetaData, error) {
-	statement := fmt.Sprintf("SELECT round_float(avg(committed::field), 2),round_float(avg(init::field), 2),round_float(max(max::field), 2),round_float(avg(used::field), 2) " +
+	statement := fmt.Sprintf("SELECT round_float(max(committed::field), 2),round_float(max(init::field), 2),round_float(max(max::field), 2),round_float(max(used::field), 2) " +
```

Comment:
I think `avg` and `max` is both ok.
Please add code links about calculating container memory graph.

Reply:
> I think `avg` and `max` is both ok. Please add code links about calculating container memory graph.

container memory graph use avg, so here is the modification consistent